src/test/ui/polymorphization/promoted-function-3.rs

```diff
@@ -0,0 +1,14 @@
+// build-pass
+// compile-flags: -Zpolymorphize=on -Zmir-opt-level=3
+
+fn caller<T, U>() -> &'static usize {
+    callee::<U>()
+}
+
+fn callee<T>() -> &'static usize {
+    &std::mem::size_of::<T>()
+}
+
+fn main() {
+    assert_eq!(caller::<(), ()>(), &7);
```

Comment:
:sweat_smile: can you change this to `run-pass` and `&0`?

r=me afterwards

Reply:
Fixed!